package.json

```diff
@@ -25,7 +25,9 @@
     "clean:libraries": "rimraf packages/**/node_modules && rimraf node_modules",
     "clean:build": "rimraf packages/**/dist && rimraf packages/**/build"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@dappnode/types": "0.1.35"
```

Comment:
you should not install this dependendcy at the root level but in the paxkage.json inside the module that requires it